config/webpack.config.prod.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const webpack = require('webpack')
+
+module.exports = {
+  output: {
+    filename: 'app.[hash].min.js'
+  },
+  devtool: '#cheap-module-source-map',
+  plugins: [
+    new webpack.optimize.DedupePlugin(),
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new webpack.optimize.UglifyJsPlugin({
+      mangle: true,
+      compress: {
+        warnings: false
+      }
+    }),
+    new webpack.DefinePlugin({
+      __SERVER__: false,
+      __DEVELOPMENT__: false,
+      __DEVTOOLS__: false,
+      __STACK_ASSETS__: true
```

Comment:
`__STACK_ASSETS__` is used in https://github.com/cozy/cozy-onboarding-v3/pull/14/commits/3d0d83ec7d6f0327320c4753fcc9bc33352e9b12#diff-c6429b99ea286602fe43ed65b7efad81R31
For the other ones, does it make sens to keep compliance cozy-template : 
https://github.com/cozy/cozy-template/blob/master/config/webpack.config.prod.js ?